include/infinispan/hotrod/ConfigurationBuilder.h

```diff
@@ -21,10 +21,22 @@ using namespace infinispan::hotrod::event;
 namespace infinispan {
 namespace hotrod {
 
+/**
+ * ClusterConfigurationBuilder is used to configure the clusters that are part of the system.
+ * Is a builder of ClusterConfiguration objects.
```

Comment:
"ClusterConfigurationBuilder is used to configure the clusters that are part of the system and is a builder of ClusterConfiguration objects."